units.libs.lua

```diff
@@ -183,7 +183,7 @@ StaticLibrary {
 		Filters = {
 			{ Pattern = "[/\\]windows[/\\]"; Config = { "win32-*", "win64-*" } },
 			{ Pattern = "[/\\]macosx[/\\]"; Config = "mac*-*" },
-			{ Pattern = "[/\\]x11[/\\]"; Config = { "x11-*" } },
+			{ Pattern = "[/\\]X11[/\\]"; Config = { "x11-*" } },
```

Comment:
that directory should be lower cased :/ I guess the correct fix here is to rename it on linux and submit that and it should work as expected.

The problem is that if the directory is called x11 on my Windows machine, that means that tundra (as the X11 isn't in the listing above) will be handled as a 'regular' directory that isn't masked and will try to compile the X11 code on Windows which will of course fail.